skrf/circuit.py

```diff
@@ -91,7 +98,7 @@
 
 from .constants import INF, S_DEF_DEFAULT, NumberLike
 from .media import media
-from .network import Network, s2s
+from .network import connect, Network, s2s
```

Comment:
```suggestion
from .network import Network, connect, s2s
```

ok sorry, preferred syntax is classes and then functions, each group ordered.